app/Resources/MgiletNotificationBundle/views/notification_dropdown.html.twig

```diff
@@ -0,0 +1,58 @@
+{% if mgilet_unseen_notification_count() > 0 %}
+<li class="btn-warning active">
```

Comment:
You should have only one `<li>` tag, and write an if condition inside this tag.

I mean, `if` should decide if show `class` attribute, not the whole `li`.